### PR TITLE
[mdanalysis] include new text selectors

### PR DESCRIPTION
**Pull request by orbeckst**
* new text selectors for definitions (dt tags) in dl (definition lists) as used in the sphinx-generated API docs (to index the function/class names themselves);
  dd (data tags) do not need a selector because text content is wrapped in p tags inside the dd and is already selected
* new text selectors for pre (code blocks)
* see issue #4699

# Pull request motivation(s)

### What is the current behaviour?

*If the current behaviour is a bug, please provide all the steps to reproduce and screenshots with context.*

### CODE REVIEW

**Comment by hmacdope** [comment]
Just so you know I'm no expert on this stuff @orbeckst. If you are happy with it then merge away. :)

**Comment by orbeckst** [comment]
I can’t merge this PR by myself. 

I need an algolia dev such as @s-pace to have a look.

**Comment by hmacdope** [comment]
I am stupid, i thought this was on a different repo.

File: configs/mdanalysis.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
     "lvl2": "[itemprop='articleBody'] > .section h3, .page h3, .post h3, .body > .section h3",
     "lvl3": "[itemprop='articleBody'] > .section h4, .page h4, .post h4, .body > .section h4",
     "lvl4": "[itemprop='articleBody'] > .section h5, .page h5, .post h5, .body > .section h5",
-    "text": "[itemprop='articleBody'] > .section p, .page p, .post p, .body > .section p, [itemprop='articleBody'] > .section li, .page li, .post li, .body > .section li"
+    "text": "[itemprop='articleBody'] > .section p, .page p, .post p, .body > .section p, [itemprop='articleBody'] > .section li, .page li, .post li, .body > .section li, [itemprop='articleBody'] > .section dt, .body > .section dt, [itemprop='articleBody'] > .section pre, .page pre, .post pre, .body > .section pre, "
```
Comment (shortcuts) [inline configs/mdanalysis.json:26]: FYI, having this number of selectors could add a lot of noise to the search results and hide other relevant results. A new crawl will be started right after the merge, don't hesitate to revert if you're not happy with the results!

Comment (orbeckst) [reply configs/mdanalysis.json:26]: Thank you for the comment @shortcuts . 

Most of the selectors were added earlier by `@s-pace` to properly index our sites. If adding dt and pre to this list creates too much noise we'll take you up on your offer to revert.

Thank you for the opportunity to try it out.

I _hope_ that this also somehow solves #4699 .

**Pull request merged**